### PR TITLE
Fix design, format and Internationalize texts in newsletter, testimonial and mailing list template

### LINKED ISSUE #80: Ensure all text is internationalized
As per the [WP.org theme requirements](https://make.wordpress.org/themes/handbook/review/required/#8-language-internationalization):

> All text strings must be translatable using gettext, with the temporary exception of text in HTML template files.

The Mailing List and Testimonial patterns contain text that has not been internationalized. There could be more strings though, so we should check the entire theme as part of this work.

It would also be good to replace "Cours" with "Course" in the string `thanks to Cours` in `testimonial.php`.

### PR DESCRIPTION
**Pull request by Imran92**
Fixes https://github.com/Automattic/course/issues/80

## Changes proposed

- Added internationalization for texts in patterns where it was missing
- Formatted the templates
- Fixed a minor spacing issue in newsletter template (the line on the left of the text was just a bit longer)

## Testing instructions

- Add the Mailing List and Testimonial patterns on a page
- Make sure they look good
- Make sure no texts are missing internationalization in these patterns
- Make sure all texts are rendering

## Mailing list

Before:
Screen Shot 2022-11-12 at 12 08 43 AM

After:
Screen Shot 2022-11-12 at 12 08 02 AM

### CODE REVIEW

File: patterns/newsletter.php [hunk 1]
```diff
@@ -7,15 +8,17 @@
 ?>
 
 <!-- wp:group {"style":{"spacing":{"blockGap":"1.25rem"}},"layout":{"type":"constrained","justifyContent":"left"}} -->
-<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"medium"} -->
-<h2 class="has-medium-font-size" style="text-transform:uppercase"><?php echo esc_html__( 'Newsletter', 'course' ); ?></h2>
-<!-- /wp:heading -->
+<div class="wp-block-group">
+      <!-- wp:heading {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"medium"} -->
+      <h2 class="has-medium-font-size" style="text-transform:uppercase"><?php echo esc_html__('Newsletter', 'course'); ?></h2>
```
Comment (donnapep) [inline patterns/newsletter.php]: Could we preserve the [PHP coding standards for whitespace](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage) in this file? Looks like they were correct before. We'll need to get a linter set up at some point to make catching violations easier.

File: patterns/testimonial.php [hunk 2]
```diff
@@ -8,31 +9,48 @@
...
-<h2 class="has-league-gothic-font-family has-x-large-font-size" style="margin-bottom:40px;line-height:1;text-transform:uppercase">“I always wanted to write, and<br>thanks to Cours, I got it right. My<br>writing is clearer, and I can finally get my message across.”</h2>
-<!-- /wp:heading -->
+        <!-- wp:column {"verticalAlignment":"bottom"} -->
+        <div class="wp-block-column is-vertically-aligned-bottom">
+            <!-- wp:heading {"style":{"spacing":{"margin":{"bottom":"40px"}},"typography":{"textTransform":"uppercase","lineHeight":"1"}},"fontSize":"x-large","fontFamily":"league-gothic"} -->
+            <h2 class="has-league-gothic-font-family has-x-large-font-size" style="margin-bottom:40px;line-height:1;text-transform:uppercase"><?php
+                echo wp_kses(
+                    __('“I always wanted to write, and<br>thanks to Cours, I got it right. My<br>writing is clearer, and I can finally get my message across.”', 'course'),
```
Comment (donnapep) [inline patterns/testimonial.php]: Whitespace violation here.

**Review by donnapep — DISMISSED** [review]
Looks good except for some whitespace coding violations. I left comments inline.

**Comment by Imran92** [comment]
Good catch @donnapep, fixed them [here](https://github.com/Automattic/course/pull/85/commits/a6697f112d83219443af434c06cd212b64afdcec). I habitually added the spaces correctly, but noticed that the default formatter of the editor was beautifying the markup but also removing the spaces. The approval got stale unfortunately :( Can you kindly approve again? Thanks

**Review by donnapep — APPROVED** [review]

**Pull request merged**